.github/workflows/ci.yml

```diff
@@ -0,0 +1,235 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should we include the release branches here @hasheddan and @jbw976?

Reply:
I believe so 👍